__tests__/app.test.js

```diff
@@ -110,4 +110,53 @@ describe("/api/articles", () => {
             expect(body.articles).toBeSortedBy("created_at", { descending: true, })
         });
     });
+})
+
+describe("/api/articles/:article_id/comments", () => {
+    it("GET 200: returns an array of comments for the given article_id when given a valid and present article_id", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.comments).toHaveLength(11)
+            body.comments.forEach(comment => {
+                expect(comment).toHaveProperty("comment_id", "votes", "created_at", "author", "body")
+                expect(comment.article_id).toBe(1)
+                expect(typeof comment.comment_id).toBe("number")
+            });
+        });
```

Comment:
This test appears a little disjointed to me.
1. you've used `.toHaveProperty` but then checked for multiple properties. this matcher needs those properties to be in an array to work : https://jestjs.io/docs/expect#tohavepropertykeypath-value
2. You've only checked the type of one of the properties.

I'd suggest refactoring this to use `.toHaveProperty`  and `expect.any` to check each property individually, and use the second argument of this matcher to check the type.
e.g. `expect(comment).toHaveProperty("comment_id", expect.any(Number))`